### ISSUE TITLE
Stub for `django.contrib.admin.sites.AdminSite._build_app_dict` is missing

### ISSUE THREAD

**Issue by AndreyMZ** [typeddjango/django-stubs#333]
# Bug report

## What's wrong

The following code produces the following warning.

`myproject/apps/admin/sites.py`:
~~~py
class MyAdminSite(admin.AdminSite):
    def get_app_list(self, request: WSGIRequest) -> List[Any]:
        app_dict = self._build_app_dict(request)
        # ...
~~~

> Unresolved attribute reference '_build_app_dict' for class 'MyAdminSite'

## How is that should be

No errors/warnings.

## System information

- OS: Windows 10
- `python` version: 3.7.4
- `django` version: 2.2.10
- `mypy` version: -
- `PyCharm` version: 2019.3.3 (Community Edition)
- `django-stubs` version: 1.4.0

**Comment by sobolevn** [comment]
@AndreyMZ will you please fix it?

**Comment by kszmigiel** [comment]
@sobolevn @AndreyMZ 
I will try to take care of this one, any tips?

**Comment by sobolevn** [comment]
@kszmigiel you can inspect django sources (including tests and docs) to find the required type for this. That's probably the only tip 😃

**Comment by AndreyMZ** [comment]
Why not release `django-stubs` 1.5.1?

**Comment by sobolevn** [comment]
We will release `1.6` soon enough.

**Comment by AndreyMZ** [comment]
It's been 6 weeks, but django-stubs 1.6 is still not released...

**Comment by AndreyMZ** [comment]
It's been almost half a year since the issue is closed, but neither django-stubs 1.5.1 nor 1.6 is released. @sobolevn, is this project still alive?

**Comment by sobolevn** [comment]
@AndreyMZ you can see that there are new issues, new bugfixes, new features merged. So, the project is 100% alive.
We also have our internal routine. And we release versions when it is time to release them.

In the meantime everyone can use:
1. `master` branch
2. specific commit hash
3. a custom fork
